blueprint/index.md

```diff
@@ -1,102 +1,114 @@
+This Genesys Developer Cloud Blueprint provides instructions for building a chat translation assistant which uses the AWS Translate service to allow customers and agents to chat in their preferred languages. The chat translation assistant automatically translates everything in the chat window in real-time, including canned responses.
 
-Genesys Cloud uses the Interaction Widget to translate and send messages to the customer.
+![Chat translation assistant](images/flowchart.png "Chat translation assistant")
 
-![Flowchart](images/flowchart.png "Flowchart")
+[toc]
 
 ## Solution Components
-* **Genesys Cloud** - The Genesys cloud-based contact center platform. Genesys Cloud is the platform for the Chat Translator solution.
-* **Genesys AppFoundry** - The Genesys AppFoundry is an app marketplace for solutions that run on the Genesys Cloud platform. You get the Chat Translator integration used in the solution from the Genesys AppFoundry.
-* **Interaction Widget Integration** - This integration type enables web apps to be embedded in an iframe within Genesys Cloud. Each iframe is unique to an interaction and will only show when the interaction is being offered or received by an agent. The client app for this blueprint will be embedded via this integration type.
-* **AWS Identity and Access Management (IAM)** - An AWS Identity and Access Management (IAM) user is an entity that you create in AWS to represent the person or application that uses it to interact with AWS. The Chat Translator uses AWS IAM to access Amazon Translate and AWS SDK.
-* **Amazon Translate** - Amazon Translate is a translation service that delivers fast, high-quality, and affordable language translation. Amazon Translate is the translate service used in the Chat Translator solution.
 
-### Software Development Kit (SDK)
-* **Genesys Cloud Platform API SDK** - This SDK is used for the initial interaction of agent and customer over chat.
-* **Amazon Web Services SDK** - This SDK enables developers to build libraries and applications that use AWS services. You can use the JavaScript API in the browser and inside Node.js applications on the server.
+* **Genesys Cloud** - A suite of Genesys cloud services for enterprise-grade communications, collaboration, and contact center management. You deploy the Chat Translator solution in Genesys Cloud.
+* **Genesys AppFoundry** - The Genesys app marketplace for solutions that run on the Genesys Cloud platform. You download the integration used in this solution from the Genesys AppFoundry.
```

Comment:
The developers can download the template files in this repository from the AppFoundry by clicking "Get It Now" on the blueprint. You may view the [Genesys Cloud Chat Assistant Blueprint](https://appfoundry.genesys.com/filter/genesyscloud/listing/7ea62ddd-e183-42ce-9ba9-ee1f37370660) from the AppFoundry that I previously authored as an example.

![image](https://user-images.githubusercontent.com/38449514/112932088-e7109680-914f-11eb-888c-a22710280cc6.png)